examples/GPR/main.jl

```diff
@@ -4,9 +4,9 @@ import NPZ
 import PyPlot
 const plt = PyPlot
 
-include("../../src/GPR.jl")
+include(joinpath("..","..","src","GPR.jl"))
 
-const gpr_data = NPZ.npzread("../../test/GPR/data/data_points.npy")
+const gpr_data = NPZ.npzread(joinpath("..","..","test","GPR","data","data_points.npy"))
```

Comment:
Maybe a minor point, but is this standard or something? don't see why the proposed way is clearer/better

Reply:
It's more OS friendly. For poor windows users (like myself) this translates to `..\\..\\test\\GPR\\data\\data_points.npy`.

Reply:
oh, I see
funnily enough, I'm myself a windows user (at the time being, hopefully not for long) and this works for me, i.e. I wrote this code :D I guess, mingw makes the difference? anyway, I'm always in for cross-platform